src/openforms/formio/tests/test_component_translations.py

```diff
@@ -0,0 +1,986 @@
+    ):
+        component = {
+            "type": "licenseplate",
+            "key": "licenseplate",
+            "label": "Must always have a label",
+            prop: f"Default {prop} value",
+            "openForms": {"translations": {lang_code: {prop: translation}}},
+        }
+
+        register.localize_component(component, lang_code, enabled=True)
+
+        self.assertEqual(component[prop], translation)
+        self.assertNotIn("translations", component["openForms"])
+
+
+class BSNTranslationTests(SimpleTestCase):
+    @given(
+        lang_code=language_code,
+        prop=st.sampled_from(["label", "description", "tooltip"]),
+        translation=st.text(min_size=1),
+    )
+    def test_translatable_properties_bsn(
+        self, lang_code: str, prop: str, translation: str
+    ):
+        component = {
+            "type": "bsn",
+            "key": "bsn",
+            "label": "Must always have a label",
+            prop: f"Default {prop} value",
+            "openForms": {"translations": {lang_code: {prop: translation}}},
+        }
+
+        register.localize_component(component, lang_code, enabled=True)
+
+        self.assertEqual(component[prop], translation)
+        self.assertNotIn("translations", component["openForms"])
+
+
+class NPFamilyMembersTranslationTests(SimpleTestCase):
+    @given(
+        lang_code=language_code,
+        prop=st.sampled_from(["label", "description", "tooltip"]),
+        translation=st.text(min_size=1),
+    )
+    def test_translatable_properties_npFamilyMembers(
+        self, lang_code: str, prop: str, translation: str
+    ):
+        component = {
+            "type": "npFamilyMembers",
+            "key": "npFamilyMembers",
+            "label": "Must always have a label",
+            prop: f"Default {prop} value",
+            "openForms": {"translations": {lang_code: {prop: translation}}},
+        }
+
+        register.localize_component(component, lang_code, enabled=True)
+
+        self.assertEqual(component[prop], translation)
+        self.assertNotIn("translations", component["openForms"])
+
+
+class SignatureTranslationTests(SimpleTestCase):
+    @given(
+        lang_code=language_code,
+        prop=st.sampled_from(["label", "description", "tooltip"]),
+        translation=st.text(min_size=1),
+    )
+    def test_translatable_properties_signature(
+        self, lang_code: str, prop: str, translation: str
+    ):
+        component = {
+            "type": "signature",
+            "key": "signature",
+            "label": "Must always have a label",
+            prop: f"Default {prop} value",
+            "openForms": {"translations": {lang_code: {prop: translation}}},
+        }
+
+        register.localize_component(component, lang_code, enabled=True)
+
+        self.assertEqual(component[prop], translation)
+        self.assertNotIn("translations", component["openForms"])
+
+
+class CoSignTranslationTests(SimpleTestCase):
```

Comment:
Do we also need to test the old cosign? Or is it really going to be deprecated?

Reply:
I've no idea, what fields are translatable on that?